### PR TITLE
Add gradient for PieChart

### PR DESCRIPTION
**Pull request by raldhafiri**
rebased pull request as per guidelines
and closing old https://github.com/imaNNeo/fl_chart/pull/1511

### CODE REVIEW

**Comment by imaNNeo** [comment]
Please update the [pie_chart.md](https://github.com/imaNNeo/fl_chart/blob/master/repo_files/documentations/pie_chart.md) docs
(read the [Contributing guideline](https://github.com/imaNNeo/fl_chart/blob/master/CONTRIBUTING.md) carefuly)

File: repo_files/documentations/pie_chart.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ When you change the chart's state, it animates to the new state internally (usin
 |:---------------|:---------------|:-------|
 |value| value is the weight of each section, for example if all values is 25, and we have 4 section, then the sum is 100 and each section takes 1/4 of the whole circle (360/4) degree|10|
 |color| colors the section| Colors.red
+|gradient| You can use any [Gradient](https://api.flutter.dev/flutter/dart-ui/Gradient-class.html) here. such as [LinearGradient](https://api.flutter.dev/flutter/painting/LinearGradient-class.html) or [RadialGradient](https://api.flutter.dev/flutter/painting/RadialGradient-class.html)|null|
```
Comment (imaNNeo) [inline repo_files/documentations/pie_chart.md:39]: Nice, please also mention that you have to provide either `color` or `gradient`

Comment (raldhafiri) [reply repo_files/documentations/pie_chart.md]: thanks, Its updated now

**Comment by imaNNeo** [comment]
Thanks for contributing!

**Pull request merged**